README.md

```diff
@@ -2,31 +2,273 @@
+itself:
+
+`public static main(String[] args)`
+
+## Default (No Access Modifier)
+
+The default access modifier is when no access modifiers are specified.
+
+Classes, fields, methods and constructors defined without an access modifier
+are available to all other classes in the same package.
+
+If we go back to our `Bicycle` class example, the variables `color` and `height`
+both use the default access level since the access modifier is not specified
+prior to the data type. This means that we can make use of those variables even
+outside the `Bicycle` class as long as it is within the same package of the
+`Bicycle` class. Again, we will discuss packages more in depth in a later
+lesson.
+
+## Private
+
+`private` is the most restrictive access modifier. Methods, fields and
+constructors defined with this modifier can only be accessed within the class
+where they are defined.
+
+The `private` modifier is very useful in hiding implementation details of a
+specific class from other classes that might use it. A very common pattern for
+doing this is to define `private` instance variables and make their values accessible
+through `public` accessor methods, which when used in this way are usually called
+"accessor methods".
+
+We'll be learning about accessor methods shortly, but for now, let us apply this knowledge
+to the variables in the `Bicycle` class as an example.  We'll update the
+fields to add the `private` keyword at the beginning of each variable declaration:
+
+```java
+public class Bicycle {
+   private String color;
+   private int height;
+}
+```
+
+The modification we made here is that we made the instance variables `color` and `height`
+private - meaning that they can only be accessed within the `Bicycle` class.
+
+So now what happens if we have the following `main` method that attempts to assign
+values to the private instance variables?
+
+```java
+public static void main(String[] args) {}
+    Bicycle johnsBike = new Bicycle();
+    johnsBike.color = "blue";
+    johnsBike.height = 36;
+}
+```
+
+If the `main` method is contained **within** the `Bicycle` class, then
+the method will compile without error because the private
+instance variables are visible to all methods in the same class.  For example,
+the code below compiles without an error:
+
+
+```java
+public class Bicycle {
+
+    private String color;
+    private int height;
+
+    public static void main(String[] args) {
+        Bicycle johnsBike = new Bicycle();
+        johnsBike.color = "blue";
+        johnsBike.height = 36;
+    }
+}
+```
+
+However, if the `main` method is  contained in a separate class, for example `TestBicycle`, we will
+see a compiler error because `TestBicycle` can't access the private members of `Bike`:
+
+<table>
+
+<tr>
+
+<td>
+<pre>
+<code>
+
+public class Bicycle {
+  private String color;
+  private int height;
+}
+
+</code>
+</pre>
+</td>
+
+<td>
+<pre>
+<code>
+
+public class TestBicycle {
```

Comment:
This looks a little awkward on GitHub's markdown preview. 
There's no code formatting and the code blocks have different sizing.

Reply:
Thanks, it should be fixed now.  I had to use spaces instead of tabs.